pkg/config/remote_config_structs.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type JVSKeyConfig struct {
+	// KeyName format: `[projects/*/locations/*/keyRings/*/cryptoKeys/*]`
+	KeyName string `firestore:"key_name,omitempty"`
```

Comment:
`firestore:"key_name,omitempty"` this part seems not required

Reply:
Or I wonder if using `json` or `yaml` would do any difference.

Reply:
If we don't write it, with firestore implementation, it will default to read the filed with name `KeyName`.

Reply:
Hmm... unfortunately this problem in firestore was not resolved https://github.com/googleapis/google-cloud-go/issues/1475.

We can keep it. But I think to make it consistent, we should add the `yaml` tag as other config types.

Reply:
In JVS, besides `yaml` tag, we also use `json` tag https://github.com/abcxyz/jvs/blob/114e40bff255c7c448a74bf2183ca64633215325/apis/v0/jwt_specification.go#L24 and `protobuf` tag https://github.com/abcxyz/jvs/blob/114e40bff255c7c448a74bf2183ca64633215325/apis/v0/cert_action_request.pb.go#L148. Even though the file is under `pkg/config` but they are used for reading/writting from firestore, not the same as other configs with `yaml` tag. I don't think it introduces incosistency.

Reply:
Those under `v0` are proto **generated**. Those are APIs and specs.

Reply:
https://github.com/googleapis/google-cloud-go/issues/1475 was talking about firestore can't do marshal and unmarshal of some specific types. I tested it, in our case, custom marshal and unmarshal works.